docs/core/diagnostics/distributed-tracing-collection-walkthroughs.md

```diff
@@ -0,0 +1,326 @@
+---
+title: Collect a distributed trace - .NET
+description: A tutorial to collect distributed traces in .NET applications
+ms.topic: tutorial
+ms.date: 03/14/2021
+---
+
+# Collect a distributed trace
+
+**This article applies to: ✔️** .NET Core 5.0 and later versions **or** any .NET application using the 
+[DiagnosticSource NuGet package](https://www.nuget.org/packages/System.Diagnostics.DiagnosticSource/5.0.1) version 5 or later
+
+.NET applications can be instrumented using the <xref:System.Diagnostics.Activity?displayProperty=nameWithType> API to produce
```

Comment:
Is the term `application` here referring to any application and library?

Reply:
Yes that was intent, but I changed 'application' -> 'code' to avoid any implication that this is only for executables.